packages/client/.eslintrc.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
     'func-names': 'off',
     'vue/multi-word-component-names': 'off',
     'vue/no-mutating-props': 'off',
+    // Allow for-of statements
+    'no-restricted-syntax': 0,
```

Comment:
Nit:

I see you removed one in-line eslint-disable comment and disabled this rule globally here.  Was that the only failing line?

If so, I think I'd just as soon leave the in-line exception in place instead of blanket disabling.

Reply:
Thanks for taking a look! 

In addition to the line that had already been excepted, there were a handful of places where this rule came up before I ran the `--fix` command / added the global disable - I would have had to have added exceptions for each of these affected lines. 

I do think `for-of` is a bit more readable overall; that said, starting with baseline compatibility with airbnb might be good and we can then make changes across the board for all files. I can re-disable for now for consistency

Reply:
Oh, you make a good point.  I wouldn't want to prevent ourselves from using for-of syntax entirely.  I definitely don't feel strongly here.